sample/build.gradle

```diff
@@ -129,20 +132,22 @@ if (isCi) {
         }
       }
 
-  tasks.withType(L8DexDesugarLibTask)
-      .matching { it.name == "l8DexDesugarLibExternalStagingAndroidTest" }
-      .configureEach {
-        doLast {
-          println "Checking expected input rules from diagnostics output"
-          String diagnosticFilePath = "build/intermediates/keeper/diagnostics/externalStagingAndroidTestMergedL8Rules.pro"
-          String diagnostics = file(diagnosticFilePath).text
-          if (!diagnostics.contains("-dontobfuscate")) {
-            throw new IllegalStateException("L8 diagnostic rules don't have Keeper's configured '-dontobfuscate', see ${diagnosticFilePath}")
-          } else if (!diagnostics.contains("-keep class j\$.time.Instant")) {
-            throw new IllegalStateException("L8 diagnostic rules include the main variant's R8-generated rules, see ${diagnosticFilePath}")
+  if (verifyL8) {
+    tasks.withType(L8DexDesugarLibTask)
+        .matching { it.name == "l8DexDesugarLibExternalStagingAndroidTest" }
```

Comment:
I _think_ that `matching {}` will realize these tasks (because `name` isn't available on the `TaskProvider`... I think) -- ah, but this is in a sample.

Reply:
Name exists on taskprovider!